src/analyze/analyze-plot.c

```diff
@@ -158,12 +159,46 @@ static void svg_graph_box(double height, double begin, double end) {
                             SCALE_Y * height);
         }
 }
+
+static void plot_tooltip(const char *name, const RelatedUnits *r) {
+        static const struct { const char *name; size_t off; } categories[] = {
+                { "After", offsetof(RelatedUnits, after) },
+                { "Before", offsetof(RelatedUnits, before) },
+                { "Requires", offsetof(RelatedUnits, after) },
+                { "Requisite", offsetof(RelatedUnits, requisite) },
+                { "Wants", offsetof(RelatedUnits, wants) },
+                { "Conflicts", offsetof(RelatedUnits, conflicts) },
+                { "Upholds", offsetof(RelatedUnits, upholds) },
+                {}
```

Comment:
Please drop the extra empty element, and use `FOREACH_ARRAY()` and/or `ELEMENTSOF()` in the below.